libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -281,6 +281,9 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM
     void update_EKF1(void);
     void update_EKF2(void);
 
+    // update of AOA and SSA estimations
+    void update_AOA_SSA(void);
```

Comment:
I expect to call these function more often than just for logging and mavlink. So If it's get called more than once between updates it seemed reasonable to make it store value and read it when needed.